### PR TITLE
Address both Yoav's concerns about the security considerations...

### PR DESCRIPTION
**Pull request by cabo**
... section and Harald's observations about privacy considerations.

### CODE REVIEW

File: draft-ietf-core-problem-details.md [hunk 1]
```diff
@@ -494,9 +495,31 @@ In this case, a new Custom Problem Detail key can simply be
...
+Concise Problem Details data item, care needs to be taken that they do
+not leak sensitive information.
+Entities storing or forwarding Concise Problem Details data items need
+to consider whether this leads to information being transferred out of
+the context within which access to sensitive information was acceptable.
+See also {{ignore-unknown}} (the last paragraph of the introduction to
+that section).
+If privacy-sensitive information is obscured in some way in the
+problem details (e.g., by base64-encoding), this might lead to
+misclassification as non-sensitive.
```
Comment (thomas-fossati) [inline draft-ietf-core-problem-details.md]: I would rephrase this as an actual recommendation:
```suggestion
Privacy-sensitive information in the problem details SHOULD
NOT be obscured in ways that might lead to misclassification as
non-sensitive (e.g., by base64-encoding).
```

**Review by thomas-fossati — APPROVED** [review]
Great stuff.  Minor editorial comment that you may ignore.

File: draft-ietf-core-problem-details.md (hunk 1 above)
Comment (cabo) [reply draft-ietf-core-problem-details.md]: redacted is not quite the word I'd use here... This is more about the original inclusion of information that is obscured.  Redacting should be done by removing, not by obscuring, but that is a different consideration.

Comment (thomas-fossati) [reply draft-ietf-core-problem-details.md]: 👀 at the dictionary: redacted can mean "obscured" but also "censored".  We only want to talk about the former here, so you are right "obscured" is the right term. (Suggested text modified accordingly).

**Comment by thomas-fossati** [comment]
🚢 it!
